core/bourbon/helpers/_lightness.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
+
+@function _gamma($channel) {
```

Comment:
We should move this function into its own partial and also add SassDoc documentation.